test/noted/notes_test.exs

```diff
@@ -19,9 +19,10 @@ defmodule Noted.NotesTest do
       note
     end
 
-    test "list_notes/0 returns all notes" do
+    test "list_notes/1 returns all notes" do
       note = note_fixture()
```

Comment:
The note_fixture should probably create the note for a specific user.

Then we can do `Notes.list_notes(note.user_id)`.